account_move_sale_partner/models/account_move.py

```diff
@@ -0,0 +1,10 @@
+# Copyright 2025 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    sale_partner_id = fields.Many2one("res.partner", readonly=True)
```

Comment:
To follow the same pattern as `partner_shipping_id`.
```suggestion
    partner_sale_id = fields.Many2one("res.partner", string="Sold-to Partner", readonly=True)
```